packages/scheduler/src/Scheduler.js

```diff
@@ -33,6 +61,12 @@ if (hasNativePerformanceNow) {
 } else {
   timeRemaining = function() {
     // Fallback to Date.now()
+    if (
+      firstCallbackNode !== null &&
+      firstCallbackNode.expirationTime < currentExpirationTime
+    ) {
+      return 0;
+    }
```

Comment:
I'm a little confused here, it seems we can just do
```
const now = hasNativePerformanceNow ? performance.now() : Date.now();
// or for tree-shaking? we can use if (hasNativePerformanceNow) { now = performance.now()} else { now = Date.now() }

timeRemaining = function() { ... } // we just need do this once rather than twice currently
```

Reply:
@bvaughn I don't understand. The code I show above **don't** "do a conditional check inside of a very "hot" function", it just do the check **once** when the `index.js` which be bundled **first run**. And it reduce duplicate code. So it's a win-win thing

Reply:
@bvaughn Sorry for the typo, I mean:
`const now = hasNativePerformanceNow ? performance.now : Date.now`, later we can just use `now()`